### DISCUSSION TITLE
Open in IntelliJ

### DISCUSSION THREAD

**Discussion by waltherjochen** [spatie/ray#361] [Ideas]
If you click on the output link you can open the file in PHPStorm, but I use IntelliJ.
This is not supported yet.

I use Windows 10

![image](https://user-images.githubusercontent.com/15231733/110906176-e08abe00-830b-11eb-9ab4-c846bd3bfe3c.png)

**Comment by hrsa** [comment]
Seems to be the same problem in the [thread 838](https://github.com/spatie/ray/discussions/838).

Ray needs to parse those links correctly
